lib/Translation/ExportVerilog/ExportVerilog.cpp

```diff
@@ -719,6 +721,27 @@ StringRef ModuleEmitter::addName(ValueOrOp valueOrOp, StringRef name) {
   return updatedName;
 }
 
+//===----------------------------------------------------------------------===//
+// SplitModuleEmitter
+//===----------------------------------------------------------------------===//
+
+namespace {
+
+class SplitModuleEmitter {
```

Comment:
Please pull this down later in the file to keep it more localized, around line 2500 or so into its own logical area clumped with the implementation logic.